orchagent/orchdaemon.cpp

```diff
@@ -193,7 +193,7 @@ bool OrchDaemon::init()
      * when iterating ConsumerMap.
      * That is ensured implicitly by the order of map key, "LAG_TABLE" is smaller than "VLAN_TABLE" in lexicographic order.
      */
-    m_orchList = { gSwitchOrch, gCrmOrch, gBufferOrch, gPortsOrch, gIntfsOrch, gNeighOrch, gRouteOrch, copp_orch, tunnel_decap_orch, qos_orch, wm_orch, policer_orch };
+    m_orchList = { gSwitchOrch, gCrmOrch, gPortsOrch, gBufferOrch, gIntfsOrch, gNeighOrch, gRouteOrch, copp_orch, tunnel_decap_orch, qos_orch, wm_orch, policer_orch };
```

Comment:
This is a better order as BufferOrch::doTask() needs to wait for PortsOrch::isInitDone() to be true

Reply:
It is interesting that BufferOrch was placed behind PortsPorch earlier as of this pr https://github.com/Azure/sonic-swss/pull/515